test/adapter/json_api/fields_test.rb

```diff
@@ -62,7 +62,7 @@ def test_fields_relationships
           end
 
           def test_fields_included
-            fields = { posts: [:author], comments: [:body] }
+            fields = { posts: [:author, :comments], comments: [:body] }
```

Comment:
With this fix, the included comments would be kicked out if we don't include them in the `fields` and thus the spec would fail.